js/kafka/utils.js

```diff
@@ -0,0 +1,4 @@
+export const sanitizeTopicName = topic => {

```

Comment:
Move to the existing file (https://github.com/ripe-tech/ripe-bus-api-js/blob/master/js/kafka/kafka-client.js), use the same way to declare a function (using `function()...` and add jsdocs.